flight/PiOS/posix/pios_serial.c

```diff
@@ -56,15 +56,19 @@ static void PIOS_SERIAL_TxStart(uintptr_t udp_id, uint16_t tx_bytes_avail);
 static void PIOS_SERIAL_RxStart(uintptr_t udp_id, uint16_t rx_bytes_avail);
 
 typedef struct {
-	int fd;
+	int readfd, writefd;
 
+	// TODO: Think about the volatility of callbacks, etc.
+	// (Not unique to this module)
 	pios_com_callback tx_out_cb;
 	uintptr_t tx_out_context;
 	pios_com_callback rx_in_cb;
 	uintptr_t rx_in_context;
 
 	uint8_t rx_buffer[PIOS_SERIAL_RX_BUFFER_SIZE];
 	uint8_t tx_buffer[PIOS_SERIAL_RX_BUFFER_SIZE];
+
+	bool dont_touch_line;
```

Comment:
What exactly does dont_touch_line mean? Seems to actually mean that the device is stdio?